apps/communications/contacts/test/unit/export/generic_export_test.js

```diff
@@ -80,6 +82,8 @@ suite('Contacts Exporter', function() {
     real_ = window._;
     window._ = navigator.mozL10n.get;
 
+    window.Loader = MockLoader;
```

Comment:
Actually you can use `MocksHelper` in [1] and you have a more readable code.

[1] https://github.com/mozilla-b2g/gaia/pull/30700/files#diff-3407ab06d83e82e2d919ccbdd7ccb3eeR49